### DISCUSSION TITLE
why this code throws 403 error?

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#644] [Q&A] [answered]
```

     BODY = {
    "expires_timestamp": "2022-09-22T02:28:11.762Z",
    "label": "string",
    "revoked": False
    }
        t=falconpy.APIHarness(client_id=client_id,client_secret=secret)
        response =   t.command("tokens_create", body=BODY)

        print(response)

```

tnx
netanel

**Comment by jshcodes** [comment, marked as answer]
Can we see the returned error message?  This code appears to work from my testing (spacing on the assignment of `response` is bad though), so I'd like to see the full result you're getting back. (A 403 is a token issue, have you checked your API credentials scope?)
